From 6adf152ab9e48ce7f646ed4bcdae6737c3418666 Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Sun, 27 Oct 2024 01:15:02 +0100
Subject: [PATCH 1/1] Enforce Solus Defaults

1. Set "always" by default for PreserveWorkingDirectory to better
   handle use of non-bash (e.g. zsh) shells
2. Set ThemeVariant to dark by default
---
 src/org.gnome.Terminal.gschema.xml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/org.gnome.Terminal.gschema.xml b/src/org.gnome.Terminal.gschema.xml
index 97f77435..57eae4bc 100644
--- a/src/org.gnome.Terminal.gschema.xml
+++ b/src/org.gnome.Terminal.gschema.xml
@@ -122,7 +122,7 @@
 
   <!-- Profiles list schema -->
 
-  <schema id="org.gnome.Terminal.ProfilesList" 
+  <schema id="org.gnome.Terminal.ProfilesList"
           extends="org.gnome.Terminal.SettingsList"
           path="/org/gnome/terminal/legacy/profiles:/">
     <override name="list">['b1dcc9dd-5262-4d8d-a863-c897e6d979b9']</override>
@@ -283,7 +283,7 @@
       <description>If true, the command inside the terminal will be launched as a login shell (argv[0] will have a hyphen in front of it).</description>
     </key>
     <key name="preserve-working-directory" enum="org.gnome.Terminal.PreserveWorkingDirectory">
-      <default>'safe'</default>
+      <default>'always'</default>
       <summary>Whether to preserve the working directory when opening a new terminal</summary>
       <description>
         Controls when opening a new terminal from a previous one carries over the working directory of the opening terminal to the new one.
@@ -701,7 +701,7 @@
     </key>
 
     <key name="theme-variant" enum="org.gnome.Terminal.ThemeVariant">
-      <default>'system'</default>
+      <default>'dark'</default>
       <summary>Which theme variant to use</summary>
     </key>
 
-- 
2.47.0

